test/functional/mempool_reorg.py

```diff
@@ -68,10 +68,8 @@ def run_test(self):
         assert_raises_rpc_error(-26, 'non-final', self.nodes[0].sendrawtransaction, timelock_tx)
 
         self.log.info("Create spend_2_1 and spend_3_1")
-        spend_2_utxo = wallet.get_utxo(txid=spend_2['txid'])
-        spend_2_1 = wallet.create_self_transfer(utxo_to_spend=spend_2_utxo)
-        spend_3_utxo = wallet.get_utxo(txid=spend_3['txid'])
-        spend_3_1 = wallet.create_self_transfer(utxo_to_spend=spend_3_utxo)
+        spend_2_1 = wallet.create_self_transfer(utxo_to_spend=spend_2["new_utxo"])
+        spend_3_1 = wallet.create_self_transfer(utxo_to_spend=spend_3["new_utxo"])
```

Comment:
Just for reference, the utxo will still stay in the MiniWallet in this test, as the tx is broadcast with the node's `sendrawtransaction`, not with the `MiniWallet.sendrawtransaction` method.

Also, the generate below doesn't reset it, for the same reason.

I guess it will be hard to prevent in practise, so it seems fine to leave as-is, unless there is a test failure?

Reply:
I would agree that is not such of a big deal, maybe until this kind of usage leads to unexpected behavior. I cannot see a way to prevent that, and neither I checked for existing or thought about future cases where usage of node's `sendrawtransaction` & `generate` might be required alongside MiniWallet usage.